src/commands/dev-env-sync-sql.js

```diff
@@ -141,6 +138,26 @@ export class DevEnvSyncSQLCommand {
 		} );
 	}
 
+	generateSearchReplaceMap() {
+		this.searchReplaceMap = {};
+
+		for ( const url of this.siteUrls ) {
+			this.searchReplaceMap[ url ] = this.landoDomain;
+		}
+
+		const networkSites = this.env.wpSitesSDS.nodes
+		if ( ! networkSites ) return;
+
+		for ( const site of networkSites ) {
+			if ( ! site.blogId || site.blogId === 1 ) continue;
+
+			const url = site.homeUrl.replace( /https?:\/\//, '' );
+			if ( ! this.searchReplaceMap[ url ] ) return;
```

Comment:
nit: I'm wondering why we're calling a `return` here. Should we have a small comment here to explain it?

Reply:
Ahh this actually is a mistake. It should be `continue` 
```suggestion
			if ( ! this.searchReplaceMap[ url ] ) continue;
```